### ISSUE TITLE
imdb plugin stopped working

### ISSUE THREAD

**Issue by y** [Flexget/Flexget#827]
After a recent update, the imdb plugin has stopped working properly. The search for the movies appears to succeed, but something in the plugin then fails to find something on the page and all movies result in  the same messages:
```
2016-01-27 20:56 DEBUG    imdb_lookup   my_movies lookup for Love the Coopers 2015 BDRip x264-GECKOS
2016-01-27 20:56 VERBOSE  imdb_lookup   my_movies Parsing imdb for `Love the Coopers 2015 BDRip x264-GECKOS`
2016-01-27 20:56 DEBUG    requests                      Fetching http://www.imdb.com/title/tt2279339/
2016-01-27 20:56 ERROR    utils.imdb    my_movies Possible IMDB parser needs updating, Please report on Github.
2016-01-27 20:56 VERBOSE  imdb_lookup   my_movies Unable to set imdb_name for tt2279339 from http://www.imdb.com/title/tt2279339/

$ flexget --version
1.2.439
You are on the latest release.
```

**Comment by crawln45** [comment]
Please read other Git issues before posting. This is a duplicate please remove it.

**Comment by y** [comment]
Link to the ticket if it's a duplicate, I read the titles of all open tickets and didn't see similar.

**Comment by crawln45** [comment]
That would be because a fix has already been merged #825

**Comment by msj33** [comment]
Working here again - thanks for quick help - Much appreciated:-)
